pubspec.yaml

```diff
@@ -18,11 +18,20 @@ dev_dependencies:
   flutter_lints: ^1.0.4
   flutter_test:
     sdk: flutter
+  test: ^1.21.1
 
-  # Specifying a slightly older version minimum for the 'pure' Dart test package
-  # because the flutter_test package needs an older version of test_api.
-  test: 1.19.5
-
+dependency_overrides:
```

Comment:
I notice you are pinning exact versions here, rather than using `^` or `>=` syntax to be less specific and support automatic upgrades.

I'm finding it difficult, on a quick Google, to find any good docs on whether that is even possible - but I did find [this video](https://www.youtube.com/watch?v=Nn6kle_DThM) which seems to suggest you can, if that would work for what you're trying to achieve. 🤔